examples/albert/run_aux.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import logging
+import os
+from dataclasses import asdict
+from pathlib import Path
+from typing import Dict, Any
+import time
+import wandb
+import torch
+import transformers
+from transformers import (set_seed, HfArgumentParser, TrainingArguments,
+                          AlbertTokenizerFast, AlbertConfig, AlbertForPreTraining)
+from transformers.optimization import get_linear_schedule_with_warmup
+from transformers.trainer_utils import is_main_process
+from torch_optimizer import Lamb
+
+import hivemind
+from arguments import CollaborationArguments, DatasetArguments, AlbertTrainingArguments
+import metrics_utils
+
+logger = logging.getLogger(__name__)
+LRSchedulerBase = getattr(torch.optim.lr_scheduler, '_LRScheduler', None)
+
+
+def setup_logging(training_args):
```

Comment:
It appears that most of this code is borrowed from `./run_trainer.py`.

Maybe we should refactor this script into an option (e.g. `--auxiliary`) for `run_trainer.py`
What do you think?


/* and optionally rename it into something like `run_peer/worker/your/ideas/here`? */